test/code/scxsystemlib/memory/memoryinstance_test.cpp

```diff
@@ -493,13 +493,148 @@ class TestWI11691MemoryDependencies : public MemoryDependencies
 };
 #endif  //defined(linux)
 
+#if defined(linux)
+class TestMemAvailableMemoryDependencies : public MemoryDependencies
+{
+    scxulong  m_totalMemory,
+        m_freeMemory,
+        m_buffers,
+        m_cached,
+        m_memAvail;
+public:
+    TestMemAvailableMemoryDependencies( scxulong  totalMemory, scxulong freeMemory,
```

Comment:
Why pass in the memory values? We have a `/proc/meminfo` file that exhibits the problem, just test this as is from the customer, and verify that the returned values are correct with the new code.